tests/supabase_filters_test.toit

```diff
@@ -0,0 +1,373 @@
+// Copyright (C) 2023 Toitware ApS.
```

Comment:
No message about the license file.

Reply:
Will update the license in all the test files in a separate PR.